webui/react/src/pages/WorkspaceList/WorkspaceCard.tsx

```diff
@@ -39,31 +40,30 @@ const WorkspaceCard: React.FC<Props> = ({ workspace, fetchWorkspaces }: Props) =
       onDropdown={onClick}>
       <div className={workspace.archived ? css.archived : ''}>
         <Row>
-          <div className={css.icon}>
+          <Column width="hug">
             <Avatar palette="muted" size={Size.ExtraLarge} square text={workspace.name} />
-          </div>
-          <div className={css.info}>
-            <div className={css.nameRow}>
-              <Typography.Title
-                className={css.name}
-                ellipsis={{ rows: 1, tooltip: true }}
-                level={5}>
-                {workspace.name}
-              </Typography.Title>
-              {workspace.pinned && <Icon name="pin" title="Pinned" />}
-            </div>
-            <p className={css.projects}>
-              {workspace.numProjects} {pluralizer(workspace.numProjects, 'project')}
-            </p>
-            <div className={css.avatarRow}>
-              <div className={css.avatar}>
+          </Column>
+          <Column>
+            <div className={css.info}>
+              <Row width={215}>
+                <Title size={TypographySize.XS} truncate={{ rows: 1, tooltip: true }}>
+                  {workspace.name}
+                </Title>
+                {workspace.pinned && <Icon name="pin" title="Pinned" />}
+              </Row>
+              <Row>
+                <Label size={TypographySize.XS}>
+                  {workspace.numProjects} {pluralizer(workspace.numProjects, 'project')}
+                </Label>
+              </Row>
+              <Row>
                 <Spinner conditionalRender spinning={Loadable.isNotLoaded(loadableUser)}>
                   {Loadable.isLoaded(loadableUser) && <UserAvatar user={user} />}
                 </Spinner>
-              </div>
-              {workspace.archived && <div className={css.archivedBadge}>Archived</div>}
+                {workspace.archived && <div className={css.archivedBadge}>Archived</div>}
```

Comment:
created a ticket to cover replacing the archived badge on Workspace and Project Cards with the hew Badge component